ouroboros-consensus/tools/db-convert/Main.hs

```diff
@@ -166,7 +166,7 @@ validateChainDb
   -> Bool -- Verbose
   -> IO ()
 validateChainDb dbDir cfg onlyImmDB verbose =
-  ResourceRegistry.with $ \registry -> give protocolMagicId $ give epochSlots $
+  ResourceRegistry.withRegistry $ \registry -> give protocolMagicId $ give epochSlots $
```

Comment:
Remove the qualifier, now we have `Registry` 3x on one line. It's almost as bad as Java 😀

Reply:
Yeah, fair enough. Just wanted to make minimal changes to stuff not directly in consensus. Will change.